src/Compilers/Core/Portable/Symbols/ITupleTypeSymbol.cs

```diff
@@ -14,7 +13,7 @@ namespace Microsoft.CodeAnalysis
     /// </remarks>
     public interface ITupleTypeSymbol : ITypeSymbol
```

Comment:
@AlekseyTs I was actually tempted to add a new `Kind` too, but it does seem this causes a lot of trouble. I'm ok to bring what I need into `INamedTypeSymbol` and remove `ITupleTypeSymbol`. This will connect well with what we discussed the other day too.

Should I just create three new members (one to check tuple type, one to get the types, one to get the members), or should I try to overload some existing members (for example `TypeArguments` could expose the types)?